compiler/llvm/clangUtil.cpp

```diff
@@ -5197,20 +5298,29 @@ static void handlePrintAsm(std::string dotOFile) {
       disSymArg += "_";
     }
 
+    // restore names to print from disk if we are in make binary-only invocation
+    if (fDriverDoMakeBinary) restorePrintIrCNames();
     std::vector<std::string> names = gatherPrintLlvmIrCNames();
-    for (const auto& name : names) {
-      printf("\n\n# Disassembling symbol %s\n\n", name.c_str());
-      fflush(stdout);
-      std::vector<std::string> cmd;
-      cmd.push_back(llvmObjDump);
-      std::string arg = disSymArg; // e.g. --disassemble=
-      arg += name;
-      cmd.push_back(arg);
-      cmd.push_back(dotOFile);
-
-      mysystem(cmd, "disassemble a symbol",
-               /* ignoreStatus */ true,
-               /* quiet */ false);
+    printf("%lu symbol names to disassemble\n", names.size());
+    if (names.empty()) {
+      USR_WARN(
+          "requested assembly dump, but no symbols to be disassembled were "
+          "specified");
+    } else {
```

Comment:
It appears the only way for this error to be encountered is if the user does specify Chapel symbols to be disassembled, but we don't find them so we don't create cname mappings for them. Otherwise, the argument parser would complain to them about missing required arguments. I'll change the wording of this error to say we couldn't find any of the symbols that were requested to be disassembled, and then leave it to https://github.com/Cray/chapel-private/issues/5387 to emit this error earlier and tell the user what symbols couldn't be found.